src/isolatable-hybrid.store.ts

```diff
@@ -2,6 +2,15 @@ import { HybridConfigurationStore, IAsyncStore, ISyncStore } from '@eppo/js-clie
 
 export type ServingStoreUpdateStrategy = 'always' | 'expired' | 'empty';
 
+/**
+ * Extension of AsyncStore where its aware it's paired with a SyncStore.
+ * Useful when providing custom persistent stores.
+ */
+export interface IPairedAsyncStore<T> extends IAsyncStore<T> {
```

Comment:
I welcome suggestions for better interface name 😅

Reply:
`RehydratingAsyncStore`